unattended_installer/install_functions/installCommon.sh

```diff
@@ -270,10 +270,63 @@ function installCommon_getPass() {
     done
 }
 
+function installCommon_installCheckDependencies() {
```

Comment:
Order functions alphabeticaly.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2089/commits/f60dfa04387b4fd109eed128b1ee3019d07659a8.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2089/commits/f60dfa04387b4fd109eed128b1ee3019d07659a8.